packages/mds-web-sockets/server.ts

```diff
@@ -105,9 +105,11 @@ export const WebSocketServer = () => {
   })
 
   const {
-    env: { NATS = 'localhost', STAN_CLUSTER = 'nats-streaming', STAN_CREDS, TENANT_ID = 'mds' }
+    env: { NATS = 'localhost', STAN_CLUSTER = 'nats-streaming', STAN_CREDS }
   } = process
 
+  const TENANT_ID = getEnvVar('TENANT_ID', 'mds')
```

Comment:
Should `getEnvVar` be used for `NATS` et al as well?

Reply:
Yup